.releaserc.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
I'm a bit curious how you came up with this config as it's quite a bit different from ours, but that's not a blocker.

Reply:
Nice catch, removed this to align better with the approach we use in PF react